src/runner/Runner_config.ml

```diff
@@ -48,6 +48,7 @@ type t = {
   (* Main flags *)
   pattern_string : string;
   pattern_file : filename;
+  pro : bool;
```

Comment:
What if we avoided adding `pro` here and we just checked that the hooks are setup? Really `pro` here can be thought as a synonym of `Option.is_some Match_tainting_mode.hook_setup_hook_function_taint_signature || Option.is_some Dataflow_tainting.hook_function_taint_signature`.

Reply:
this is true. i wanted to avoid it because i thought that this line would look more "clean", and also because i was afraid of a future where we have some pro features that don't involve the hooks, but it's probably a safe bet, so I don't mind changing it